flytekit/models/admin/workflow.py

```diff
@@ -1,4 +1,6 @@
 import typing
+from datetime import datetime as _datetime
```

Comment:
can we not use the `_library` pattern anymore.  Let's make this just `from datetime import datetime`

Reply:
I'll just hide that column completely, thanks for the suggestion. Since the version is a list and is shown when you add the task as a subcommand